### ISSUE TITLE
【バグ】 "Danger level 5" "Dump Witness" quest ambushes newbie with much higher level monsters

### ISSUE THREAD

**Issue by smbhax** [hengband/hengband#2472]
## 環境 (environment)

- OS: Cygwin in Windows 10, Curses front end
- バージョン: 3.0.0Alpha57 + latest code, English version

## バグの詳細 (bug details)

The first quest you get from the inn in the starting town, Outpost, is "Dump Witness," whose description says it is "Danger level: 5." However, this quest ambushes the player with a level 13 mimic type monster, a cloaker, who, particularly since they can paralyze, can kill a low-level character before they even see it--that's what happened to my 100 HP, AC 25, level 6 weaponsmith.

Video of that encounter here: https://youtu.be/bbgOEMpc2pY?t=5032

The other monster in the quest is a level 9 monster, a killer bee: a very fast monster who may burst upon the player from a glass-enclosed room.

Both of these creatures seem well over "danger level" of 5, if "danger level" is meant to correspond at all to monster/character level (if it isn't, this needs to be explained clearly to the player!).

"Dump Witness" is quite likely to be the first quest a new player attempts in the game, and hitting them with two monsters who could kill them instantly seems like a horrible introduction to the game. It made me think that maybe I shouldn't try quests at all, since--based on this initial experience--it seems they may just murder me with an overpowered monster.

## 再現手順 (repro steps)

1) In the starting town of Outpost, go to the inn ("+" on the southeast side of town) and request a quest--if it's your first request there, it will probably be the "Dump Witness" quest, describing itself as "Danger level: 5"
2) Proceed to the yellow ">" quest entrance southwest of the inn and enter the quest area there
3) Walk around the west side of the area and enter through the regular door at the southwest
4) Enter the main room
==> The "(" in the room is a cloaker, a level 13 monster who strikes from ambush with paralysis. The "I" in the adjoining, glass-enclosed room is "moves quickly" poisonous level 9 monster. Both of these can kill a level 5 character rather easily, making the quest feel distinctly unbalanced for a level 5 character--if that is how "danger level" is meant to be read...and that is how it WILL be read by a new player, in the absence of other information (and there is no help file on quests).

この手順を実行する事でバグを発生させる事ができるセーブファイルがあれば添付 (attach save files)

Level 8 weaponsmith in Outpost:
[197609.zip](https://github.com/hengband/hengband/files/8411289/197609.zip)

## 期待される正常な動作 (expected behavior)

A danger level 5 quest should be challenging but not outright deadly to a level 5 player--which means it should probably not contain monsters of significantly higher level, possessing deadly hit properties (poison, paralysis), who can strike without warning and kill the character instantly.

This is probably the first quest a new player will attempt and it should break the newbie in somewhat gently so that they have a reasonable chance to learn the game's systems without being forced to start over.

If this is a misunderstanding of what "danger level" means, then danger level needs to be explained clearly to the new player before they undertake a quest.

## スクリーンショット (screenshot)
![cloaker](https://user-images.githubusercontent.com/91744319/161585370-ec766ac8-3812-4536-bde3-def56ccfb2c0.jpg)

**Comment by dis-** [comment]
Mimics are NOT move, so player can easily defeat them by long-range attacks.
The game has a myriad of enemies that can easily bury the unprepared player.
This quest is responsible for conveying the "die and learn" balance of the entire game.
"danger level" 5 is intended.

**Comment by smbhax** [comment]
Is the intent to teach the player to run some sort of enemy detection spell/ability/ability when entering a quest area? I'm not too keen on having to use detection stuff constantly. Not that my level 6 character had any such capability yet, at any rate. I suppose what this is teaching me is to avoid quests and stick to random dungeons.

**Comment by dis-** [comment]
If you walk carefully with a lantern, you can notice the mimic before it attacks you.
Using a sling, light crossbow, or throwing a torch, you can easily defeat them. You can purchase them in town.
The quest message will indicate that "something is hiding in the house", so any character who unintentionally steps into the darkness will not be able to complain about death.

**Comment by smbhax** [comment]
Oh, it wasn't dark; I had just bought a lantern = ) and was wielding it there, so the room was illuminated; the cloaker never showed up in my monster list subwindow even when I moved right next to it and dropped dead; it just looked like a "(" cloak item. This is all visible in the screenshot. My level 6 weaponcrafter saw the "cloak" clearly, but never registered it as a monster. Possibly I failed some sort of detection roll there?

(In retrospect I was also wondering why my high-elf's infravision didn't spot the body heat coming from the "cloak," but I think maybe infravision doesn't function with that sort of detail in the game.)

The one curious thing that I see playing my recording back is that it had changed to ( from an isolated gray "x" mark--no other "x" marks around it-- when I got halfway across the room, which might be a little unusual--I've become used to mostly ignoring the "not searched for traps" "x" marks though, so that didn't jump out at me at the time.

Anyway the last two times I've died have been on quests so I think I'll avoid them until I'm better at the game. : )
